pkg/bootstrap/docker/up.go

```diff
@@ -714,14 +714,19 @@ func (c *ClientStartConfig) EnsureDefaultRedirectURIs(out io.Writer) error {
 
 // CheckAvailablePorts ensures that ports used by OpenShift are available on the Docker host
 func (c *CommonStartConfig) CheckAvailablePorts(out io.Writer) error {
+	userNamespace, err := c.DockerHelper().UserNamespaceEnabled()
```

Comment:
Initial idea was to invoke this function during helpers creation but in this case we couldn't report about possible error. The only was that I found is to invoke it from `CheckAvailablePorts()`. That's why signature of `TestPorts()` has been modified.